aspnetcore/blazor/fundamentals/additional-scenarios.md

```diff
@@ -190,21 +235,22 @@ To adjust the reconnection retry count and interval:
 </body>
 ```
 
-### Hide or replace the reconnection display
+## Hide or replace the reconnection display
```

Comment:
We recently added a feature that allowed a user to customize the delay before the reconnection display would show up. We might want to document this here as well. Users can modify this by setting the `transition-delay` property in the CSS for the modal element.

```
'transition: visibility 0s linear 500ms'
```

The default is above which delays the display by 500ms second. Users can modify this value to delay it.